source/instructors/setupcourses/assignment-settings/penalties.rst

```diff
@@ -10,6 +10,8 @@ You can add penalties to the assignment if students don't complete it within the
 
 Penalties are not cumulative. Your student will receive the penalty associated with their submission date.
 
+To apply Penalties, a **Due Date** for the assignment must be set (see :ref:`Assignment Duration <assignment-duration>`), penalties will be applied beginning on the due date.
+
 .. Note:: It is recommended that you set the closing date/time for the assignment (see :ref:`Assignment Duration <assignment-duration>`) to the last date/time for the penality to avoid having to change the assignment status for a student who submits the assignment late. 
```

Comment:
This note can go away - we have removed the confusion of needing to set the closing date beyond the last penalties date.